packages/objectsender/src/examples/browser/main.ts

```diff
@@ -118,3 +127,33 @@ class RandomFoo extends Base {
     this.noise = Math.random().toString(16)
   }
 }
+
+class RandomJoe extends Base {
+  @Detach()
+  @Chunkable(10)
+  numbers: number[]
+
+  constructor(props?: Record<string, unknown>) {
+    super(props)
+    this.numbers = times(100, () => 42)
+  }
+}
+
+export class Collection<T extends Base> extends Base {
```

Comment:
Nice! I think some of this in the future can be slowly extracted out to canonical speckle types proper, and exported out by the lib.